dynawo/sources/Simulation/DYNSimulation.h

```diff
@@ -563,7 +563,7 @@ class Simulation {
    */
   bool hasIntermediateStateToDump() const;
 
- public:
+ private:
```

Comment:
Are you sure this does not break dynawo-algorithms build?

Reply:
no I can test it, but I hope it does not because it would mean that we do some things very ugly there...

Reply:
tested it and fixed it